Mathlib/Data/Nat/Init.lean

```diff
@@ -434,6 +434,44 @@ protected lemma div_left_inj (hda : d ∣ a) (hdb : d ∣ b) : a / d = b / d ↔
   refine ⟨fun h ↦ ?_, congrArg fun b ↦ b / d⟩
   rw [← Nat.mul_div_cancel' hda, ← Nat.mul_div_cancel' hdb, h]
 
+/-- TODO: Replace `Nat.sub_mul_div` in core. -/
+lemma sub_mul_div' : (a - b * c) / b = a / b - c  := by
```

Comment:
Arguments should be explicit unless mentioned in later arguments, since it's a rewriting lemma:
```suggestion
lemma sub_mul_div' (a b c : ℕ) : (a - b * c) / b = a / b - c := by
```

Reply:
Added explicit variables in the lemmas where not all of them are mentioned in the hypothesis.